kamu-core/src/infra/reset_service_impl.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use crate::domain::*;
+use std::sync::Arc;
+use dill::*;
+use opendatafabric::*;
+
+
+pub struct ResetServiceImpl {
```

Comment:
At this point it is not obvious. If we are going to have more switches, i.ex., hard reset, soft reset, and other functions that enrich "reset" behavior, it would be clear argument to keep it as a service. If it stays as current, yes, it can be simplified as an operation on the aggregate.